src/index.html

```diff
@@ -11,22 +11,46 @@
       content="ie=edge"
     />
     <title>Document</title>
+
     <link
       rel="stylesheet"
       href="style.css"
     />
   </head>
+
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div
+      class="search-wrapper"
+      tabindex="0"
+    >
+      <img
+        class="search-icon"
+        src="images/Search.svg"
+        alt="Search icon"
+      />
+      <input
+        class="search field-large"
+        type="text"
+        data-qa="keypress big"
+        placeholder="Try 'Los Angeles'"
+      />
+    </div>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <div
+      class="search-wrapper"
+      tabindex="0"
+    >
+      <img
+        class="search-icon"
+        src="images/Search.svg"
+        alt="Search icon"
+      />
+      <input
+        class="search field-small"
+        type="text"
+        data-qa="hover small"
+        placeholder="Try 'Los Angeles'"
+      />
+    </div>
```

Comment:
There are two div elements with the class 'search-wrapper' containing similar content. Ensure that this duplication is intentional. If not, consider removing or modifying one of them to avoid redundancy.